public/zoho-chat.css

```diff
@@ -12,7 +12,7 @@
 .zsiq_floatmain.siq_bR div.zsiq_cnt {
   height: 70px;
   background: #fff;
-  padding: 14px 18px;
+  padding: 14px 16px;
```

Comment:
Mmm, it fixes the issue, but if the language changes, it wil be broken again.

Is it not possible doing something more dynamic?